src/fidgets/ui/IFrame.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import TextInput from "@/common/components/molecules/TextInput";
+import { FidgetArgs, FidgetProperties, FidgetModule } from "@/common/fidgets";
+import { isValidUrl } from "@/common/lib/utils/url";
+import useSafeUrl from "@/common/lib/hooks/useSafeUrl";
+
+export type IFrameFidgetSettings = {
+  url: string;
+};
+
+const DISALLOW_URL_PATTERNS = [
```

Comment:
This is the same as `UNSAFE_URL_PATTERNS` in `src/common/lib/hooks/useSafeUrl.ts`. Maybe move this to a constants file?

Reply:
Or just drop it from here, it looks like the second arg to `useSafeUrl` is optional

Reply:
I repeated it here to make it explicit for anyone viewing this file, and to avoid issues if the `useSafeUrl` file defaults are changed